lighthouse/config/schemas.py

```diff
@@ -11,6 +11,25 @@
         "type": "string",
         "check_with": "plate_events_dependent_parameters",
     },
+    "user_id": {
+        "required": True,
+        "type": "string",
+    },
+    "robot": {
+        "required": True,
+        "type": "string",
+    },
+    "event_wh_uuid": {
```

Comment:
What is this UUID?

Reply:
Is the uuid of the event. If the event fails or is updated, we can use the same uuid to rebroadcast the changes in the warehouse and update it.